react/features/base/config/functions.any.js

```diff
@@ -49,6 +49,19 @@ export function getMeetingRegion(state: Object) {
     return state['features/base/config']?.deploymentInfo?.region || '';
 }
 
+/**
+ * Selector used to get feature flags.
+ *
+ * @param {Object} state - The global state.
+ * @param {string} featureFlag - The name of the feature flag.
+ * @returns {boolean}
+ */
+export function getFeatureFlags(state: Object, featureFlag) {
```

Comment:
that's good, but I expect us having to check on the `SOURCE_NAME_SIGNALING` specifically in many places, can you add a convenience selector for the source name signaling on top of this one?

Reply:
we can also add a convenience selector later, doesn't need to be in this PR

Reply:
I've missed this comment, just updated